src/GraphQL/GraphQL.php

```diff
@@ -481,3 +483,15 @@ function annotated(array $class_names, array $with_types = [], array $with_direc
     $deannotator = new Deannotator($with_types, $with_directives);
     return $deannotator->deannotate($class_names);
 }
+
+/**
+ * Adds rule to list of global validation rules
+ *
+ * @param array<ValidationRule> $validationRules
+ */
+function addValidationRules(array $validationRules): void
```

Comment:
Awesome, thank you. Just a small thing. Siler API uses `snake_case` notation for function names and I think you can be even more concise by removing the `add` word. What you think about just `validations_rules`, namespaced it will make lot of sense like:
```php
use Siler\GraphQL;

GraphQL\validation_rules($rules);
```
More declarative then imperative, what you think?

Reply:
Yeah, totally forgot about snake case, I will fix it in couple of minutes.

But how should I fix Psalm errors? Looks like they came from previous PR... Psalm doesn't know that if variable x passed is-callable check, then this variable is callable...? Should I add another var annotation inside?

Reply:
Don't worry, I will take care of Psalm, then you can see how it looks like for further contributions.

Reply:
Cool! I pushed changes already.